pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
just look at the failed builds - I guess SparsityRecompileTest is a good example. However, note that we cannot modify these tests because people need to be able to runs the entire testsuite offline on some older laptop as well.